src/main/java/uk/gov/hmcts/cmc/claimstore/services/FormaliseResponseAcceptanceService.java

```diff
@@ -155,7 +155,7 @@ private void formaliseCCJ(Claim claim, ResponseAcceptation responseAcceptation,
             countyCourtJudgment,
             claim.getExternalId(),
             authorisation,
-            true);
+            false);
```

Comment:
Making this false means the default CCJ which will try to enforce rule that DefendantResponse is not present. And will throw `ForbiddenActionException` in `uk.gov.hmcts.cmc.claimstore.rules.CountyCourtJudgmentRule#assertCountyCourtJudgementCanBeRequested()`